src/app/controllers/myinfo.server.controller.ts

```diff
@@ -58,7 +59,7 @@ export const addMyInfo: RequestHandler<ParamsDictionary> = async (
       // Step 3: Hash the values and save them
       .andThen((prefilledFields) => {
         form.form_fields = prefilledFields
-        ;(req as WithForm<typeof req>).form = form
+        ;(req as WithJsonForm<typeof req>).form = form
```

Comment:
do we really need the semicolon here?

Reply:
yes, because without the semicolon the compiler will read it as 

```ts
form.form_fields = prefilledFields(req as WithJsonForm<typeof req>).form = form
```

which is invalid